charts/sn-platform/templates/broker/broker-cluster.yaml

```diff
@@ -192,6 +192,12 @@ spec:
     imagePullSecrets:
 {{ toYaml .Values.global.imagePullSecrets | indent 6 }}
     {{- end }}
+  {{- if .Values.tls.broker.enabled }}
```

Comment:
`tls.enabled` should also be checked here.

Reply:
Sure